### ISSUE TITLE
Highcharts plotOptions dataLabels enabled true Problems

### ISSUE THREAD

**Issue by dardochaves** [wkhtmltopdf/wkhtmltopdf#4912]
**wkhtmltopdf version(s) affected**: 
wkhtmltopdf 0.12.6 (with patched qt) 

**OS information**
Amazon Linux AMI release 2018.03

**Description**
Dear Support,

I have problems to build a pdf document when I use the Highcharts library option plotOptions -> dataLabels -> enabled: true. wkhtmltopdf does not working when this property is setted. If I use enabled: false, I don't have problems to generate the pdf document form the site or url. When wkhtmltopdf is loading pages from console, I get this error: Loading pages (1/6)Segmentation fault==========>                                ] 46% and nothing happens.

**How to reproduce**
![console](https://user-images.githubusercontent.com/30182970/102794350-05fdd080-438a-11eb-88c4-2ec4b86b3f67.png)
![enabled-true](https://user-images.githubusercontent.com/30182970/102794490-380f3280-438a-11eb-9a98-fb7c1621f6bd.png)

  <!DOCTYPE html>
  <html lang='en-US'>

  <head>
    <title>Programación Semanal | Argos Energía</title>
    <link rel="stylesheet" href="https://www.highcharts.com/demo/static/bootstrap.css" type="text/css">
    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
    <style type="text/css">
      .container {
        width: 100%
      }

      #hs-component {
        background-color: inherit;
        padding: 0
      }

      #hs-component .container {
        padding: 0;
        width: 100%
      }

      .demo {
        /*background-color: #eee;
      padding: 40px 50px*/
      }
    </style>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="Description" content="Highcharts Demo: Basic line">

    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
    <script src="https://www.highcharts.com/demo/static/bootstrap.js" defer></script>
    <script src="https://www.highcharts.com/demo/static/modernizr.js" type="text/javascript"></script>

    <script src="https://code.highcharts.com/highcharts.js"></script>
    <script src="https://code.highcharts.com/modules/series-label.js"></script>
    <script src="https://code.highcharts.com/modules/exporting.js"></script>
    <script src="https://code.highcharts.com/modules/export-data.js"></script>
    <script src="https://code.highcharts.com/modules/accessibility.js"></script>
    <script src="https://code.highcharts.com/modules/annotations.js"></script>
  </head>

  <body>
    <div id="hs-component">
      <div class="container">
        <div id="wrap" class="sidebar-wrapper">
          <div class="col-lg-9 col-md-9 col-sm-9 col-xs-12 sidebar-eq demo">
            <div class="chart-container">
              
              <script type="text/javascript">
                Highcharts.setOptions({
                        lang: {
                          thousandsSep: '.',
                          decimalPoint: ',' 
                      }
                  });
              </script>

              
              <div style="margin-top:30px;margin-bottom: 65px; width:1400px;">
                <div id="logo" style="float:right;">
                  
                </div>
              </div>

              <div style="font-size: 55px; font-weight: 600; float:left; width: 1400px; padding-left: 130px; 
                        margin-bottom: 30px; margin-top: 30px;">
                <span>TEST</span>
              </div>

              
              <div id="test" style="width:1400px;">
                
                 
                <div style="float:left;width:50%;margin-top: 0px;">
                  <style>
                    .highcharts-figure-test,
                    .highcharts-data-table table {
                      min-width: 700px;
                      max-width: 700px;
                      /* margin: 1em auto;*/
                    }
                  </style>

                  <figure class="highcharts-figure-test">
                    <div id="container"></div>
                  </figure>

                  
                  <script type="text/javascript">
                    Highcharts.chart('container', {
                      xAxis: {
                          categories: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
                      },

                      **plotOptions: {
                          series: {
                              dataLabels: {
                                  enabled: true
                              }
                          }
                      },**

                      series: [{
                          data: [29.9, 71.5, 106.4, 129.2, 144.0, 176.0, 135.6, 148.5, 216.4, 194.1, 95.6, 54.4]
                      }]
                  });
                  </script>
                </div>
                
              </div>
            </div>

          </div>
        </div>
      </div>
  </body>
  </html>
**Expected behavior**
The pdf document is generated correcctly whitout errors. 

**Possible Solution**

**Comment by dardochaves** [comment]
> @dardochaves did you find a solution for this?

@anirudhchronus No. I did not find a solution using wkhtmltopdf 0.12.6 (with patched qt).

**Comment by anirudhchronus** [comment]
@dardochaves My problem was solved by using useHTML: true. Try it

**Comment by dardochaves** [comment]
> @dardochaves My problem was solved by using useHTML: true. Try it

@anirudhchronus Genius! It works! Thanks a lot for your help! Have a nice day.
